src/App.js

```diff
@@ -1,24 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
+import Home from './components/Home';
+import Details from './components/Details';
+import './css/App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="App">
```

Comment:
- [ ] Great job with the design. Please, kindly follow the two grid colors . The first and second, like in the original design. You can remove the third. Remember, you are reproducing someone's design and following it to the fullest matters. This will help you learn more
- [ ] please also make the state/city names in the home page bolder like in the original design.
yours
![Screenshot (1091)_LI](https://github.com/rushemaa/metrics-webapp/assets/65298048/254dd114-4c56-4cb6-b739-b0b5aa503858)
original
![Screenshot (1092)_LI](https://github.com/rushemaa/metrics-webapp/assets/65298048/b9e0398a-18eb-44c3-9620-565beb5930f7)